ftw_tools/models/utils.py

```diff
@@ -0,0 +1,191 @@
+import re
+
+import geopandas as gpd
+import numpy as np
+import shapely
+from fiboa_cli.parquet import create_parquet
+
+
+def merge_polygons(
```

Comment:
This function seems a bit complicated but it's just finding overlapping pairs and then computing IoU and containment metrics and then merging any that exceed some threshold into a single polygon. This is because when patching with overlap we get duplicate predictions in the overlap regions that aren't filtered by NMS.